src/cloneCommand.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  JupyterFrontEnd,
+  JupyterFrontEndPlugin
+} from '@jupyterlab/application';
+import { ITranslator, nullTranslator } from '@jupyterlab/translation';
+import { CommandIDs, IGitExtension, Level } from './tokens';
+import { IFileBrowserFactory } from '@jupyterlab/filebrowser';
+import { Dialog, showDialog } from '@jupyterlab/apputils';
+import { GitCloneForm } from './widgets/GitCloneForm';
+import { logger } from './logger';
+import {
+  addFileBrowserContextMenu,
+  IGitCloneArgs,
+  Operation,
+  showGitOperationDialog
+} from './commandsAndMenu';
+import { GitExtension } from './model';
+import { addCloneButton } from './widgets/gitClone';
+
+export const gitCloneCommandPlugin: JupyterFrontEndPlugin<void> = {
+  id: '@jupyterlab/git:command-clone:plugin',
```

Comment:
Let's use a simpler name (especially as it gathers all clone related code):

```suggestion
  id: '@jupyterlab/git:clone',
```